map_manager_node:
  ros__parameters:
    use_predef_landmask: true
    landmask_file: "src/vortex-asv/mission/map_manager/params/land_polygon.yaml"
    map_resolution: 0.2
    map_width: 1000
    map_height: 1000
    frame_id: "world"
